pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2528,6 +2528,26 @@
         %rejected  ``noun+!>((~(has in rejected.cors-registry) u.origin))
       ==
     ::
+        [%scry @ @ ~]
+      =/  host  `i.t.tyl
+      =/  url  i.t.t.tyl
+      =/  server  (per-server-event [eny *duct now rof] server-state.ax)
+      =/  [=action suburl=@t]
+        (get-action-for-binding:server host url)
+      =/  invalid  (error-page 400 %.y url "invalid scry: {<action>} {<suburl>}")
+      :^  ~  ~  %noun
+      !>  ^-  octs
+      ?+    -.action  !!
+        %gen
+      invalid
+        %scry
+      (handle-scry %.y (need host) url)
```

Comment:
```suggestion
      ?+    -.action  !!
          %gen
        invalid
      ::
          %scry
        (handle-scry %.y (need host) url)
      ==
```

Alternatively, if they're gonna stay this short:

```suggestion
      ?+  -.action  !!
        %gen   invalid
        %scry  (handle-scry %.y (need host) url)
      ==
```

(With the caveat about not crashing of course, but this looks WIP anyway.)